src/redash_stmo/utils.py

```diff
@@ -0,0 +1,7 @@
+from redash.handlers import api
+
+def add_resource(app, resource, endpoint):
+    # After api.init_app() is called, api.app should be set by Flask (but it's not) so that
```

Comment:
Good idea making this available for more extensions. Please make that a triple-doublequotes docstring please.

Also, would you mind renaming the module to `resources.py`?